docs/3.0rc/api-ref/rest-api/server/concurrency-limits-v2/read-concurrency-limit-owners.mdx

```diff
@@ -0,0 +1,3 @@
+---
+openapi: post /api/v2/concurrency_limits/active_owners
```

Comment:
this might need to get remove manually, not sure how the integration works

Reply:
Weird, I'm not sure why that didn't automatically update. 🤔

Reply:
I'll just delete it. The correct file is there anyway and linked in the docs.